application/backend/src/main/resources/apps/typerefinery/web_root/dark.css

```diff
@@ -3,8 +3,10 @@
   --primary-text-color:#fff;
   --subtitle-color: rgb(188, 188, 188);
   --primary-object-background-color:#001E3C;
-  --ticks-color: #5D7183;	 
-  --grid-color: #5D7183;	  
-  --label-color: #ebedeb;
-  --border-color:#ebedeb;
+  --ticks-color:green;
```

Comment:
I have updated them, please pull development.

Reply:
Here,Just resolve the conflicts